packages/webapp/src/components/Map/PreviewPopup/index.jsx

```diff
@@ -55,7 +66,7 @@ export default function PurePreviewPopup({ location, history, sensorReadings, st
   const classes = useStyles();
   const { t } = useTranslation();
   const { units } = useSelector(userFarmSelector);
-  const { reading_types } = useSelector(sensorReadingTypesByLocationSelector(location.id));
+  const { reading_types = [] } = useSelector(sensorReadingTypesByLocationSelector(location.id));
```

Comment:
why it's location.id not location.location_id

Reply:
This was added by sein. I don't know why the variable name is kept as id